snooty/n.py

```diff
@@ -197,7 +197,7 @@ class SubstitutionReference(InlineParent):
 class Root(Parent[Node]):
     __slots__ = ("options",)
     type = "root"
-    options: Dict[str, str]
+    options: Dict[str, Any]
```

Comment:
I really don't want `Any` in the node fields: once we hit the AST, everything should be safely typed. Can we refine this type?